src/udev/udev-node.c

```diff
@@ -192,7 +197,7 @@ static int link_update(sd_device *dev, const char *slink, bool add) {
         _cleanup_free_ char *target = NULL, *filename = NULL, *dirname = NULL;
         char name_enc[PATH_MAX];
         const char *id_filename;
-        int r;
+        int i = 0, r = 0;
```

Comment:
No, please don't do unused initiailizations like that. Both `r` and `i` are unconditionally initialized before first use below.

Reply:
Fixed.